core/src/test/java/org/pentaho/platform/dataaccess/datasource/api/MetadataServiceTest.java

```diff
@@ -147,12 +150,11 @@ public void testGetMetadataDatasourceIds() throws Exception {
 
   @Test
   public void testGetMetadataDatasourceIdsError() throws Exception {
```

Comment:
I would say the name of the function is misleading. There aren't any errors thrown by the code here. It is just checking that the `MetadataService.getMetadataDatasourceIds` can handle the case that there are no ids returned from metadataDomainRepository.getDomainIds